docs/install/_system_requirements.rst

```diff
@@ -8,7 +8,7 @@ Operating systems
 
  - Windows Vista, 7, 8.1, 10
  - Debian/Ubuntu: 14.04, 16.04 and up - anything that's *not* end-of-life.
- - Linux: Any system with Python 2.7
+ - Linux: Any system with Python 3.4
```

Comment:
It should say "Any system with Python 2.7 and 3.4+". The issue is that the Debian installer is only built for Python3, which in any case is available on the systems listed above.